src/components/MerchantPayout/VoucherStatusModal/VoucherStatusModal.tsx

```diff
@@ -88,27 +89,35 @@ export const VoucherStatusModal: FunctionComponent<VoucherStatusModal> = ({
   const isVisible = checkValidityState === "CHECKING_VALIDITY";
```

Comment:
On line 43 there is a "Redeemed on ..." that we missed out

Reply:
Shouldn't this part be in the other PR? The "Redeemed on..." is part of the dynamic title that is displayed based on the previous transaction time, and not an error dialog.

Reply:
Can you check if it has been included? Matthea's PR is already approved